app/controllers/refine/inline/criteria_controller.rb

```diff
@@ -93,12 +93,17 @@ def clear
     handle_filter_update()
   end
 
-  private
+  def merge_groups
+    @criterion = Refine::Inline::Criterion.new(criterion_params.merge(refine_filter: @refine_filter)) 
+    Refine::Filters::BlueprintEditor
+    .new(@refine_filter.blueprint)
+    .change_conjunction(criterion_params[:position].to_i - 1, "and")
 
-  def set_blank_filter
```

Comment:
This wasn't being used anywhere and was leftover from my work on the clear button. So removed